docs/tutorial/index.rst

```diff
@@ -0,0 +1,270 @@
+
+.. note::
+
+   Our commercial site offers some extra features,
+   like support for private projects.
+   You can learn more about :doc:`our two different sites </choosing-a-site>`.
+
+First steps
+-----------
+
+Importing the project to Read the Docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To import your GitHub project to Read the Docs,
+first click on the "Import a Project" button on your dashboard
+(or browse to `the import page <https://readthedocs.org/dashboard/import/>`_ directly).
+You should see your GitHub account under the "Filter repositories" list on the right.
+If the list of repositories is empty, click the "refreshing your accounts" link,
+and after that all your repositories will appear on the center.
+
+.. figure:: /_static/images/tutorial/rtd-import-projects.png
+   :width: 80%
+   :align: center
+   :alt: Import projects view
+
+   Import projects view
+
+Locate your ``rtd-tutorial`` project
+(possibly clicking "next ››" at the bottom if you have several pages of projects),
+and then click on the "+" button to the right of the name.
+The next page will ask you to fill some details about your Read the Docs project:
+
+Name
+  The name of the project. It has to be unique across all the service,
+  so it is better if you prepend your username,
+  for example ``astrojuanlu-rtd-tutorial``.
+
+Repository URL
+  The URL that contains the sources. Leave the automatically filled value.
+
+Repository type
+  Version control system used, leave it to "Git" (which should be the default).
+
+Default branch
+  Name of the default branch of the project, leave it to ``main``.
+
+Edit advanced project options
+  Leave it unchecked, we will make some changes later.
+
+After hitting the "Next" button, you will be redirected to the project home.
+You just created your first project on Read the Docs!
+
+.. figure:: /_static/images/tutorial/rtd-project-home.png
+   :width: 80%
+   :align: center
+   :alt: Project home
+
+   Project home
+
+Checking the first build
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs will try to build the documentation of your project
+right after you create it.
+To see the build logs,
+click on the "Your documentation is building" link on the dashboard,
+or alternatively navigate to the "Builds" page,
+then open the one on top (the most recent one).
+
+If the build has not finished yet by the time you open it,
+you will see a spinner next to a "Installing" or "Building" indicator,
+meaning that it is still in progress.
+
+.. figure:: /_static/images/tutorial/rtd-first-successful-build.png
+   :width: 80%
+   :align: center
+   :alt: First successful documentation build
+
+   First successful documentation build
+
+When the build finishes, you will see a green "Build completed" indicator,
+the completion date, the elapsed time,
+and a link to see the corresponding documentation.
+If you now click on "View docs", you will see your documentation live!
+
+.. figure:: /_static/images/tutorial/rtd-first-light.png
```

Comment:
Wish the ad wasn't so prominent here, alas. Maybe a place to note Gold can remove ads?

Reply:
👍🏽